README.md

```diff
@@ -6,168 +6,208 @@ level: 100
 
 ![Build badge](https://identitydivision.visualstudio.com/_apis/public/build/definitions/a7934fdd-dcde-4492-a406-7fad6ac00e17/514/badge)
 
 ## Scenario
 
-This sample shows how to build a .NET Core MVC Web app that uses OpenID Connect to sign in personal accounts (including outlook.com, live.com, and others) as well as work and school accounts from any company or organization that has integrated with Azure Active Directory. It leverages the ASP.NET Core OpenID Connect middleware.
+This sample shows how to build a .NET Core MVC Web app that uses OpenID Connect to sign in personal accounts (including outlook.com, live.com, and others) as well as work and school accounts from any company or organization that has integrated with Azure Active Directory The Web App then  calls the Microsoft Graph on behalf of the signed-in user. The sample leverages the ASP.NET Core OpenID Connect middleware and [MSAL.NET](http://aka.ms/msalnet)
 
-![Sign-in with Azure AD](ReadmeFiles/sign-in.png)
-
-An on demand video was created for the Build 2018 event, featuring this scenario and this sample. See the video [Building Web App Solutions With Authentication](https://channel9.msdn.com/Events/Build/2018/THR5001), and the associated [PowerPoint deck](http://video.ch9.ms/sessions/c1f9c808-82bc-480a-a930-b340097f6cc1/SigninworkandschoolMSAusersWebApp.pptx)
+> This is the second tutorial, building on top of the previous one (in the master branch) showing how to add sign-in to a Web application. This ones adds the part which is about calling the Microsoft Graph in a controller on behalf of the user
 
 ## How to run this sample
 
 To run this sample:
 
-> Pre-requisites: Install .NET Core (for example for Windows) by following the instructions at [.NET and C# - Get Started in 10 Minutes](https://www.microsoft.com/net/core). In addition to developing on Windows, you can develop on [Linux](https://www.microsoft.com/net/core#linuxredhat), [Mac](https://www.microsoft.com/net/core#macos), or [Docker](https://www.microsoft.com/net/core#dockercmd).
+> Pre-requisites: go through how to run this sample from the master branch. This page only shows the increment.
 
 ### Step 1: Register the sample with your Azure AD tenant
 
-1. Sign in to the [Application registration portal](https://apps.dev.microsoft.com/portal/register-app) either using a personal Microsoft account (live.com or hotmail.com) or work or school account.
-1. Give a name to your Application, make sure that the *Guided Setup* option is **Unchecked**. Then press **Create**. The portal will assign your app a globally unique *Application ID* that you'll use later in your code.
-1. Click **Add Platform**, and select **Web**.
-1. In the Redirect URLs field, add `http://localhost:5000/` and `http://localhost:5000/signin-oidc`
+When you have registered your app as described in [the first tutorial](Step 1: Register the sample with your Azure AD tenant), you need an extra step:
```

Comment:
Is this supposed to be link?

Reply:
I think so? I added a link back to the README in the master branch.